api/runtime/cell/v0/cell.proto

```diff
@@ -29,68 +29,10 @@
 \* -------------------------------------------------------------------------- */
```

Comment:
I like that the services are broken into smaller files. Should there be a higher level runtime file? If we have shared messages, where would we put those?

Reply:
the issue i have here is that each broken down part now has its own version, which i don't think is what we want.. we want aurae.<version>.<service> (where service is runtime or runtime.cells, or whatever).

having aurae.<service>.<version> implies that we might have runtime.cells.v1 and observe.v0 which i would like to avoid.

Reply:
Definitely agree with the version being in the wrong spot. I'm also inclined to agree with your comment overall ([more context](https://github.com/aurae-runtime/aurae/pull/226#issuecomment-1368728945)).

My only reason for liking the multiple files is that smaller files are easier to grasp, but that is not a good reason to break things up if it causes other issues.

Reply:
I posted a follow-up on the structure, https://github.com/aurae-runtime/aurae/pull/226#issuecomment-1368992622.